jobs/dcusr.hcl

```diff
@@ -0,0 +1,47 @@
+job "dcusr" {
+  datacenters = ["aperture"]
+
+  type = "service"
+
+  group "dcusr" {
+    count = 1
+
+    network {
+      port "http" {
+        to = 3000
```

Comment:
I can't see the dockerfile for the image (private repo) but this appears to me as if it's not serving static content, instead it's running the dev server with `npm run start` or similar.

Reply:
Yes that's right, it isn't a static site. The site is built and then runs a node server.
The `Dockerfile` is adapted from [here](https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile).